README.md

```diff
@@ -0,0 +1,276 @@
+# Staff Graded Assignment XBlock
+
+This package provides an XBlock for use with the edX platform which
+provides a staff graded assignment. Students are invited to upload files
+which encapsulate their work on the assignment. Instructors are then
+able to download the files and enter grades for the assignment.
+
+Note that this package is both an XBlock and a Django application.
+
+## Installation
+
+### Devstack installation (for local development)
+
+Since this involves changing version-controlled files in `edx-platform`, it's a good idea to create
+a branch in that repo for all SGA development. **All of the edits below refer to `edx-platform` files or
+configuration files in devstack. No files in the `edx-sga` repo need to be edited to install and use
+the SGA XBlock in devstack.**
+
+1. **Configure file storage**
+
+    For file storage, SGA uses the same file storage configuration as other
+    applications in edX, such as the comments server.
+    devstack defaults to local storage, and fullstack defaults to
+    S3. Using local file storage is the easiest option for hacking on and debugging SGA locally.
+
+    To configure SGA to use local storage, edit `lms/envs/private.py`
+    to include these settings (and create the file if it doesn't exist yet):
+
+    ```
+    MEDIA_ROOT = "/edx/var/edxapp/uploads"
+    DATA_DIR = MEDIA_ROOT
+    DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+    ```
+    
+    You can also configure S3 to be used as the file storage backend. Ask a fellow developer or devops for the
+    correct settings to enable this.
```

Comment:
Can we link to this edX article on configuring data storage? https://openedx.atlassian.net/wiki/spaces/OpenOPS/pages/112001105/Configuring+Data+Storage